addons/mail/static/src/js/services/chat_manager.js

```diff
@@ -295,9 +297,11 @@ var ChatManager =  AbstractService.extend({
                     displayed_author: info.displayed_author || info.email_from,
                 };
                 info.name = info.record_name || info.subject || info.displayed_author;
-                info.image_src = info.module_icon || info.avatar_src;
+                info.image_src = info.module_icon || info.avatar_src || '/mail/static/src/img/smiley/mailfailure.jpg';
```

Comment:
This is a redundant information, not very linked to the message. I decided to put it in js to avoid to send it on each mail failure but we could decide to return it with _format_mail_failures

Reply:
Well I prefer the server to send complete data so that JS does not have to make some assumption on the code. Especially if we add stuff managed by the notification area, with inheritance, .. It is not a very heavy addition to the sent data size.